### PR TITLE
chat functionality

### PR DESCRIPTION
**Pull request by MohamedAmine-C**

### CODE REVIEW

File: app/Http/Controllers/MessageController.php [hunk 1]
```diff
@@ -14,17 +17,27 @@ class MessageController extends Controller
      */
     public function index()
     {
-        //
-    }
+        $id                = Auth::id();
+        $user              = User::find($id);
```
Comment (CaddyDz) [inline app/Http/Controllers/MessageController.php]: Why not simply?
`$user = Auth::user();`  
It's much faster to get the user from the session than to actually run a SQL query

File: config/database.php [hunk 2]
```diff
@@ -50,7 +50,7 @@
             'charset' => 'utf8mb4',
             'collation' => 'utf8mb4_unicode_ci',
             'prefix' => '',
-            'strict' => true,
+            'strict' => false,
```
Comment (CaddyDz) [inline config/database.php]: `MySQL 5.6` detected 😝   
Upgrade to `5.7` or simply use [Vagrant Homestead](laravel.com/docs/5.5/homestead)

File: app/Http/Controllers/MessageController.php (hunk 1 above)
Comment (MohamedAmine-C) [reply app/Http/Controllers/MessageController.php]: i agree

File: config/database.php (hunk 2 above)
Comment (MohamedAmine-C) [reply config/database.php]: lol i hate when i have query errors for no logical purpose (at least in my view).

Comment (CaddyDz) [reply config/database.php]: It's either going to be [Heroku](https://www.heroku.com/) or a custom [VPS](https://en.wikipedia.org/wiki/Virtual_private_server) and both can support bleeding edge software.

**Pull request merged**